pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>kronenthaler</groupId>
+    <artifactId>libai</artifactId>
```

Comment:
The [main class](https://maven.apache.org/shared/maven-archiver/examples/classpath.html#Make) is not set, it seems like a good idea if it runs the examples when excecuted.